manifests/deployment-config.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       serviceAccountName: peribolos-as-services
       containers:
         - name: peribolos-as-services
-          image: quay.io/harshad16/peribolos-as-service:latest
+          image: peribolos-service-controller:latest
```

Comment:
@tumido should i change our image name in quay to peribolos-service-controller instead of peribolos-as-service?

Reply:
I don't really mind either name, both is OK for me, you decide :slightly_smiling_face:

Reply:
ack, I will make it the same as the GitHub repo name, so it is easy to be referred.